src/arch/x86/lib/cpu_performance/cpu_info.h

```diff
@@ -1,13 +1,15 @@
 #ifndef X86_CPU_INFO_H_
 #define X86_CPU_INFO_H_
 
+struct cpu_feature {
+	char name[13];
+	unsigned int val; 
+};
+
 struct cpu_info {
 	char vendor_id[13];
-	unsigned int base_freq;
-	unsigned int max_freq;
-	unsigned int FPU;
-	unsigned int SSE;
-	unsigned int MMX;
+	struct cpu_feature feature[5];
```

Comment:
Same here, use macro to define array size.

Reply:
Done in the latest fixup commit!